pkg/splunk/enterprise/configuration.go

```diff
@@ -196,6 +196,33 @@ func GetSplunkService(cr enterprisev1.MetaObject, spec enterprisev1.CommonSpec,
 	return service
 }
 
+// validateVolumes initializes optional properties in Volumes to default values
+func validateVolumes(spec *enterprisev1.CommonSplunkSpec) {
```

Comment:
The name "validateVolumes" seems to be misleading as we are setting default values to the spec objects. Something like setDefaults or initDefaults ?

Reply:
I will change it to setVolumeDefaults()